### PR TITLE
Italian language files

### PR DESCRIPTION
**Pull request by stefano-buscaglia**
Romana,
I put all italian translation file online, please review it.

### CODE REVIEW

**Comment by RomanaBW** [comment]
Hi Stefano,
it looks very well, although I have no clue of the Italian language.
What about Your name as author or at least co-author?

**Comment by stefano-buscaglia** [comment]
Hi Romana,
it's ok my name as author or co-author, as are you prefer, thank you. Tell me and I do the modifications

**Comment by RomanaBW** [comment]
You're the author by my understanding

**Comment by RomanaBW** [comment]
Ciao Stefano,
have You switched on debug language files at Joomla!? This will help You find such problems…

File: src/administrator/language/it-IT/it-IT.plg_vmuserfield_bwpm_buyer2subscriber.sys.ini [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+PLG_BWPOSTMAN_PLUGIN_BUYER2SUBSCRIBER="BwPostman Plugin Buyer2Subscriber"
+PLG_BWPOSTMAN_PLUGIN_BUYER2SUBSCRIBER_DESCRIPTION="BwPostman Plugin Buyer2Subscriber è un plugin per iscriversi automaticamente a mailing list selezionabili mentre un compratore si registra con Virtuemart. <br /> <br /> Questo plugin funziona in insieme al plugin User2Subscriber, che pertanto è anche installato. <br /> <br /> La mailing lists a cui abbonarsi e il messaggio prima dell'abbonamento sono selezionabili separatamente dal plugin User2Subscriber. Le altre impostazioni del plugin User2Subscriber verranno utilizzate anche in questo plugin. <br /> <br /> Le impostazioni di visualizzazione e le impostazioni necessarie per cognome, nome, genere e campo aggiuntivo vengono utilizzati dal componente BwPostman"
+
+PLG_BWPOSTMAN_PLUGIN_BUYER2SUBSCRIBER_OPTION_DEBUG_LABEL="Debug"
+Utilizza questa opzione con attenzione! Se questa casella di controllo è selezionata, nel file di registro sono stati aggiunti molti più messaggi. Può accadere anche che i messaggi appaiano in frontend. Di conseguenza, disattivalo, se tutto funziona bene e i visitatori possono raggiungere questo sito web
```
Comment (RomanaBW) [inline src/administrator/language/it-IT/it-IT.plg_vmuserfield_bwpm_buyer2subscriber.sys.ini]: Missing language variable?

**Comment by stefano-buscaglia** [comment]
Changed author + some typo

File: src/administrator/language/it-IT/it-IT.plg_vmuserfield_bwpm_buyer2subscriber.sys.ini (hunk 1 above)
Comment (stefano-buscaglia) [reply src/administrator/language/it-IT/it-IT.plg_vmuserfield_bwpm_buyer2subscriber.sys.ini]: Add the missed language string

**Pull request merged**